### PR TITLE
Assignment 2: FD2School 02 - HTML Technology Spike

### PR DESCRIPTION
**Pull request by infantlikesprogramming**
__Pull Request Description__

This exercise on FD2School 02 - HTML Technology Spike #4 is completed (Dung Ha).
Addresses#4
---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

**Comment by braughtg** [comment]
Don't forget to link your PR to the issue in the Issue tracker.  See question number 8 in the activity.

**Comment by infantlikesprogramming** [comment]
Thank you so much for mentioning that. I have added "Addresses#4" to the text body of the pull request.

File: farmdata2/farmdata2_modules/fd2_school/HTML/html.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <hr>
+    <h1>My Sample Harvest Report</h1>
+    <p>Details: </p>
+    <ul>
+        <li><b>Farm:</b>Sample Farm</li>
+        <li><b>User:</b>manager1</li>
+        <li><b>Language:</b>English</li>
+    </ul>
+    <br>
+    <form>
```
Comment (braughtg) [inline farmdata2/farmdata2_modules/fd2_school/HTML/html.html:47]: This form tag does not seem necessary.

**Comment by braughtg** [comment]
FD2School 01 - HTML Spike Feedback:

* Overall
  - [x] - added HTML sub-tab to the FD2 School Tab (4, 5, 6)
  - [x] - Code in `.module` file is neat
  - [x] - Made PR with descriptive body. (8)
  - [x] - PR Body contains Link to the issue. (8)
  - [x] - Commits for nameable units of work. 
  - [x] - Uses descriptive commit messages.
  - [x] - PR Converted from draft to regular (30)

* HTML Sub-tab contents:
  - [x] - HTML code is neatly formatted.
  - [x] - Headers (14, 15)
  - [x] - All Details (farm, user, language, start, end, crop) (14, 15)
  - [x] - Italics, horizontal rules and line breaks (17, 18)
  - [x] - Form Elements (crop, area, date) (25, 26)
  - [x] - Table (headers, 2+ data rows) (28-29)

* Comments:
  * Assignment is complete so this PR has been closed.
  * Your score will be posed in Moodle.

**Pull request closed without merging**